payments-core/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -1794,6 +1796,48 @@ class Stripe internal constructor(
         }
     }
 
+    /**
+     * Retrieve a list of possible brands for the given card number.
+     * Returns an error if the cardNumber length is less than 6 characters.
+     *
+     * @param cardNumber the card number
+     * @param callback a [ApiResultCallback] to receive the result or error
+     */
+    @UiThread
```

Comment:
I'm surprised we have this annotations on these methods. I don't see a reason why they should be called from the UI thread. Do you have more context here?

Reply:
I do not, I can remove it